server/src/main/java/org/opensearch/extensions/ExtensionsManager.java

```diff
@@ -326,25 +326,18 @@ private void loadExtension(Extension extension) throws IOException {
                     extension.getUniqueId(),
                     // placeholder for ephemeral id, will change with POC discovery
                     extension.getUniqueId(),
-                    extension.getHostName(),
+                    extension.getHostAddress(),
```

Comment:
I think the line below this should be the port?

Reply:
I think this was confusing. 
ExtensionNode extends implementation of DiscoveryNode[1].
The constructor needs `hostName` and `hostAddress`.  Though when I looked through the codebase, hostname is not really resolved/used at this point. 

For extensions we can do:
- Continue to use hostName.
- Use hostAddress and send it to hostName.

Thinking about it I'm leaning on towards using the `hostName` WDYT @dbwiddis 

[1] https://github.com/opensearch-project/OpenSearch/blob/3e3b1551114b1d1fdb30b40bc7a38749ff1ecb3b/server/src/main/java/org/opensearch/cluster/node/DiscoveryNode.java#L124

Reply:
Good question, fixed the order and removed unnecessary params.